examples/benchmarking/private/bench_runner.rs

```diff
@@ -0,0 +1,21 @@
+use std::process;
+fn main() {
+    // `rust_benchmark` targets produce two rust binaries. We want
+    // to get the benchmark runner executable
+    let bin = env!("BENCH_FILES")
```

Comment:
Does this mean we compile the runner once per benchmark? Could this use `std::env::var` and get this path at runtime rather than compile time?

Reply:
I think it's difficult to define runtime variables like this. That's kinda what the `//units/launcher` is meant to handle but would require some work to make it more generically consumable. Which could be done but figured that could happen in another PR where we want to optimize this rule. What do you think?